tests/test_run_inputs.py

```diff
@@ -0,0 +1,129 @@
+from typing import Optional
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import (
+    ElementNotVisibleException,
+    ElementNotSelectableException,
+    NoSuchElementException,
+)
+from selenium.webdriver import ActionChains
+
+web = "https://machinist.flapflap.io/"
+run_inputs_path = "#__next > main > div.MenuBar__Root-sc-1th5wld-0.jDqDxJ.App__FloatingMenuBar-gv0e21-2.eZsxUV > ul > li:nth-child(4)"
```

Comment:
This is a really gnarly selector...

Would you be able to refactor this into a selector that grabs the `<li>` containing text `Run inputs` or whatever the text is for that button?